doc/configuration.rst

```diff
@@ -1741,6 +1742,49 @@ flag is passed to ``sphinx-build``. This can be enabled by setting::
     }
 
 
+.. _recommend_examples:
+
+Enabling the example recommender system
+=======================================
+
+Sphinx-Gallery can be configured to generate content-based recommendations for
+the example gallery. A list of related examples is automatically generated by
+computing the closest examples in the TF-IDF space of their text contents. Then,
+the most similar content is displayed at the bottom of each example as a set of
+thumbnails.
+
+The recommender system can be enabled by setting ``enable`` to ``True``. To
+configure it, pass a dictionary to the ``sphinx_gallery_conf``, e.g.::
+
+    sphinx_gallery_conf = {
+        ...
+        "recommender": {"enable": True, "n_examples": 5, "min_df": 1, "max_df": 1.0},
```

Comment:
I think we should recommend to use better values for `min_df` and `max_df`, e.g. `min_df=3` and `max_df=0.9`.

This should help decrease noisy recommendation and reduce CPU and memory usage to compute the recommendations.